src/arch.rs

```diff
@@ -136,6 +170,10 @@ registers!(Arm, {
     D31 = (287, "D31"),
 },
 aliases {
+    SP = (13, "SP"),
+    LR = (14, "LR"),
+    PC = (15, "PC"),
```

Comment:
I'm not familiar with ARM, but would it make sense for these to be the primary names? binutils objdump defaults to `-M reg-names-std` which uses these.

Reply:
I considered making them primary names since they're more common, but I wasn't sure if I should change the behavior of `register_name` so I made them aliases. I slightly prefer R13-R15 as aliases but either's fine with me.

Reply:
Going to leave it since I'm not sure if this constitutes a breaking change and I probably want to do a minor release soon.